lib/author.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/book'
+class Author 
+  attr_reader :books
+
+  def initialize(name_hash)
+    @name = {
+      first_name: name_hash[:first_name],
+      last_name: name_hash[:last_name]
+    }
+    @books = []
+  end
+
+  def name
+    "#{@name[:first_name]} #{@name[:last_name]}"
+  end
+
+  def write(title, date)
+    return false if date.length < 4
```

Comment:
Neat catch; we didn't specifically ask for this, though, and you don't test for a condition where this would get a `false` response. Also, we generally would want to return the same "kind" of data. In this case, since we'd want a `Book` object returned, we'd want a `nil` here instead of a `false`